### DISCUSSION TITLE
Creation of Custom Groups

### DISCUSSION THREAD

**Discussion by Dheemant6** [DefectDojo/django-DefectDojo#9915] [General]
How to create custom groups for imported findings from trivy ?  I am not getting any option to create group through bulk edit as mentioned

**Comment by mtesauro** [comment]
It should work the same for any tool that DefectDojo can import:

Select some findings:
![image](https://github.com/DefectDojo/django-DefectDojo/assets/3422419/99f8393e-7a30-4f3f-be6b-3d0a3c9c78dc)

Use bulk edit:
![image](https://github.com/DefectDojo/django-DefectDojo/assets/3422419/b65460df-de8d-42ac-8ccf-5f9ecf54e8ff)

Now you have a finding group:
![image](https://github.com/DefectDojo/django-DefectDojo/assets/3422419/5e0d1a9b-ae44-4952-8df3-b81286c2f27e)
